packages/react/src/GoogleLoginButton/GoogleLoginButton.tsx

```diff
@@ -25,10 +25,7 @@ export default function GoogleSSOButton(props: {
 }) {
   const { workspace, database } = props;
   const { basePath } = useNileConfig();
-  // workspace and database can be `''`.
-  // let this fail silently for now, and update the context when the time comes
-  const contextHref = `${basePath}/workspaces/${workspace}/databases/${database}/${LOGIN_PATH}`;
-  const href = props?.href ?? contextHref;
+  const href = props?.href ?? buildDefaultHref(basePath, workspace, database);
```

Comment:
I would keep this inline, since it will statically get created in the build, vs calling a function